### PR TITLE
Add API version and context to runtime metadata object

### PR DESCRIPTION
**Pull request by mpmunasinghe**
Fixing https://github.com/wso2-enterprise/choreo/issues/6473
Sample deployment object 
 `    {
        "apiFile": "60e5a80b2006786ab020b887-60e5a80e2006786ab020b888",
        "environments": [
          {
            "name": "Default",
            "vhost": "localhost"
          }
        ],
        "version": "1.0.0",
        "apiContext": "/pizzashack/1.0.0"
      }`

### CODE REVIEW

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/gatewayartifactsynchronizer/RuntimeArtifactGeneratorUtil.java [hunk 1]
```diff
@@ -69,10 +69,18 @@ public static RuntimeArtifactDto generateMetadataArtifact(String apiId, String n
             throws APIManagementException {
 
         List<APIRuntimeArtifactDto> gatewayArtifacts = getRuntimeArtifacts(apiId, gatewayLabel, tenantDomain);
+        Map<String, String> contextUuidMap = new HashMap<>();
         if (gatewayArtifacts != null) {
+
+            if (StringUtils.isNotEmpty(apiId)) {
+                contextUuidMap.put(apiId, gatewayArtifactsMgtDAO.retrieveAPIContextFromApiId(apiId));
+            } else {
+                contextUuidMap = gatewayArtifactsMgtDAO.retrieveAllAPIContext();
```
Comment (malinthaprasan) [inline components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/gatewayartifactsynchronizer/RuntimeArtifactGeneratorUtil.java:78]: Do we need to get all API's contexts from here? Can we pass the same parameters apiId, gatewayLabel, organization to a method and retrieve a map of APIInfo objects?

Comment (mpmunasinghe) [reply components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/gatewayartifactsynchronizer/RuntimeArtifactGeneratorUtil.java:78]: With this particular rest API we do not use the organization query as a filter.

File: components/apimgt/org.wso2.carbon.apimgt.internal.service/src/gen/java/org/wso2/carbon/apimgt/internal/service/RuntimeMetadataApi.java [hunk 2]
```diff
@@ -1,52 +1,49 @@
...
+import org.wso2.carbon.apimgt.internal.service.RuntimeMetadataApiService;
 import org.wso2.carbon.apimgt.internal.service.impl.RuntimeMetadataApiServiceImpl;
+import org.wso2.carbon.apimgt.api.APIManagementException;
 
-import javax.ws.rs.GET;
-import javax.ws.rs.HeaderParam;
-import javax.ws.rs.Path;
-import javax.ws.rs.Produces;
-import javax.ws.rs.QueryParam;
+import javax.ws.rs.*;
```
Comment (Rajith90) [inline components/apimgt/org.wso2.carbon.apimgt.internal.service/src/gen/java/org/wso2/carbon/apimgt/internal/service/RuntimeMetadataApi.java:8]: Shall we remove * imports

File: components/apimgt/org.wso2.carbon.apimgt.internal.service/src/gen/java/org/wso2/carbon/apimgt/internal/service/RuntimeMetadataApi.java [hunk 3]
```diff
@@ -1,52 +1,49 @@
...
 import javax.ws.rs.core.Context;
 import javax.ws.rs.core.Response;
+import javax.ws.rs.core.SecurityContext;
+import javax.inject.Inject;
 
+import io.swagger.annotations.*;
```
Comment (Rajith90) [inline components/apimgt/org.wso2.carbon.apimgt.internal.service/src/gen/java/org/wso2/carbon/apimgt/internal/service/RuntimeMetadataApi.java:14]: Shall we remove * imports

**Review by malinthaprasan — APPROVED** [review]

**Pull request merged**